Makefile

```diff
@@ -5,12 +5,10 @@ BUNDLE_IMG ?= quay.io/kiegroup/kogito-cloud-operator-bundle:$(VERSION)
 # Default catalog image tag
 CATALOG_IMG ?= quay.io/kiegroup/kogito-cloud-operator-catalog:$(VERSION)
 # Options for 'bundle-build'
-ifneq ($(origin CHANNELS), undefined)
+CHANNELS=alpha,1.x
 BUNDLE_CHANNELS := --channels=$(CHANNELS)
```

Comment:
so we will always define the `BUNDLE_CHANNELS` and `BUNDLE_DEFAULT_CHANNEL` right ?

Reply:
should those values be always the same as in `annotations.yaml` and `bundle.Dockerfile` ?

Reply:
annotation.yaml and bundle.Dockerfile are autogenerated files, defining these values edit those two files

Reply:
I defined the same channels we currently have in the OLM, and was present in the package.yml, we can review later about what channels we should offer